src/usecases/check-user-figma-auth-use-case.ts

```diff
@@ -2,11 +2,18 @@ import {
 	figmaService,
 	FigmaServiceCredentialsError,
 } from '../infrastructure/figma';
+import { ConnectInstallation } from '../domain/entities';
 
 export const checkUserFigmaAuthUseCase = {
-	execute: async (atlassianUserId: string): Promise<boolean> => {
+	execute: async (
+		atlassianUserId: string,
```

Comment:
/nit I think we could just accept a `ConnectUserInfo` param here?

Reply:
Technically, we can but I'm afraid that it can inconsistent with other use cases. Many use cases accept the entire `ConnectIntallation` (since it is automatically fetched by middleware for use). This use case contract allows to avoid an extra query to the database for use cases that require a `ConnectIntallation` (e.g., to call Jira API).

Therefore, it can be more consistent to leave it as is and consistently accept a `ConnectIntallation` in all our use cases (but it is, definitely, not a hard rule).

Please, let me know if you have concerns about this, Justin.